### PR TITLE
mediawiki: Make vue rules only apply to vue files

### PR DESCRIPTION
**Pull request by jdforrester**

### CODE REVIEW

File: mediawiki.json [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
+  },
+  "overrides": [
+    {
+      "files": [ "**/*.vue" ],
+      "extends": [
+        "plugin:vue/recommended",
+        "./vue-es5.js"
+      ],
+      "plugins": [ "es" ],
+      "parserOptions": {
```
Comment (catrope) [inline mediawiki.json]: You shouldn't need to touch parserOptions here, vue-es5.js already does this

File: mediawiki.json [hunk 2]
```diff
@@ -1,21 +1,45 @@
...
+        "rules": {
+          "vue/html-indent": [ "warn", "tab" ],
+          "vue/html-closing-bracket-newline": "off",
+          "vue/max-attributes-per-line": [ "warn", {
+            "singleline": 2,
+            "multiline": {
+              "max": 1,
+              "allowFirstLine": true
+            }
+          } ],
```
Comment (catrope) [inline mediawiki.json]: The rules above this point (html-indent, html-closing-bracket-newline, and max-attributes-per-line) should already be inherited from vue-common.json (which vue-es5.js indirectly inherits from)

**Review by catrope — CHANGES_REQUESTED** [review]

File: mediawiki.json (hunk 1 above)
Comment (jdforrester) [reply mediawiki.json]: Ah, yeah, was just hacking locally until it worked.

File: mediawiki.json [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
+    "mediawiki/valid-package-file-require": "error"
+  },
+  "overrides": [
+    {
+      "files": [ "**/*.vue" ],
+      "extends": [
+        "plugin:vue/recommended",
+        "./vue-es5.js"
+      ],
+      "plugins": [ "es" ],
```
Comment (catrope) [inline mediawiki.json]: This one is also duplicated from vue-es5.js, sorry for missing this the first time around

**Review by catrope — APPROVED** [review]

**Pull request merged**